Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm

```diff
@@ -37,6 +37,7 @@
 #if USE(APPLE_INTERNAL_SDK)
 #import <WebKitAdditions/RemoteLayerTreePropertyApplierInteractionRegionAdditions.mm>
 #else
+static Class getInterationRegionLayerSubclass() { return [CALayer class]; }
```

Comment:
WebKit coding style says this function should not have the word `get` in it. I would name it `interactionRegionLayerClass`. Need to rename it anyway, since it has the “interation” typo. in it.